jobs/jobs.go

```diff
@@ -48,5 +56,12 @@ func Start() {
 		logger.Errorf("Failed to schedule job for cleaning up job history table: %v", err)
 	}
 
+	if config.UpstreamConf.IsFilled() {
+		refTime := time.Now().Add(time.Minute * -20)
```

Comment:
@moshloop refTime will eventually come from the database. Will need to create a new table to store all the upstream push times.